packages/runtime/runtime-definitions/src/summary.ts

```diff
@@ -19,6 +19,12 @@ export interface ISummaryStats {
     totalBlobSize: number;
 }
 
+export interface ITopLevelSummaryStats extends ISummaryStats{
```

Comment:
Should we rename this to IDataStoreSummaryStats instead of ITopLevelSummaryStats?

Reply:
I am not sure about that. This logs more than just data store stats, i.e., current summary stats + data store stats.

Reply:
So maybe ISummaryStatsWithDataStoreStats? because TopLevel does not mean much imo.

Reply:
`IGeneratedSummaryStats`? Since this is returned as part of `IGeneratedSummaryData` in `generateSummary`.